shared/event/subscription.go

```diff
@@ -122,13 +124,15 @@ type resubscribeSub struct {
 	waitTime, backoffMax time.Duration
 }
 
+// Unsubscribe unsubscribes from subscription.
```

Comment:
:D

Reply:
https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo

Reply:
What can I say, it is a simple method to unsubscribe from the subscription :)